universe/ShipDesign.h

```diff
@@ -614,10 +614,9 @@ class FO_COMMON_API PredefinedShipDesignManager {
 
 private:
     PredefinedShipDesignManager();
-    ~PredefinedShipDesignManager();
 
-    std::map<std::string, ShipDesign*>  m_ship_designs;
-    std::map<std::string, ShipDesign*>  m_monster_designs;
+    std::map<std::string, std::unique_ptr<ShipDesign>>  m_ship_designs;
```

Comment:
if not lining up with other variable names, no need for the extra space before m_